lang/typescript/src/extended-address/generateAddress1.ts

```diff
@@ -0,0 +1,25 @@
+import {concatAddressField} from '../utils/address-fields';
+import {getRegionConfig} from '../utils/regions';
+import {Address} from '../types/address';
+
+/**
+ * Generate a concatenated address1 string based on the region specified by country code
```

Comment:
Maybe  _based on the region of the given address_ ?  This comment sounds like we're passing in a country code parameter, which was true in an earlier iteration of this code.